subworkflows/ebi-metagenomics/rrna_extraction/main.nf

```diff
@@ -0,0 +1,42 @@
+
+include { INFERNAL_CMSEARCH           } from '../../../modules/ebi-metagenomics/infernal/cmsearch/main'
+include { CMSEARCHTBLOUTDEOVERLAP     } from '../../../modules/ebi-metagenomics/cmsearchtbloutdeoverlap/main'
+include { EASEL_ESLSFETCH             } from '../../../modules/ebi-metagenomics/easel/eslsfetch/main'
+
+workflow RRNA_EXTRACTION {
+
+    take:
+    ch_fasta // channel: [ val(meta), [ fasta ] ]
+
+    main:
```

Comment:
In motus-pipeline we have chunking fasta file before cmsearch. Should we think about it here as well? Or will subwf work if I get chunked fasta as input (I guess no, as deoverlap requires a full/merged output from cmsearch).
It it just a thought.. I don't know would it work fast on huge fastas.

Reply:
good point - we would need to chunk before cmsearch and then merge them again before deoverlap..especially because cmsearch can be so slow on big files, definitely something to think about